<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="9" ignored="0" total="12" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="197" started-at="2017-06-27T08:54:57Z" finished-at="2017-06-27T08:54:57Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="197" started-at="2017-06-27T08:54:57Z" finished-at="2017-06-27T08:54:57Z">
      <class name="com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest">
        <test-method status="FAIL" signature="Team_getActiveMembers_should_return_a_list_of_team_members_that_are_active()[pri:0, instance:com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest@643b1d11]" name="Team_getActiveMembers_should_return_a_list_of_team_members_that_are_active" duration-ms="6" started-at="2017-06-27T14:24:57Z" finished-at="2017-06-27T14:24:57Z">
          <exception class="java.lang.ClassCastException">
            <message>
              <![CDATA[java.lang.String cannot be cast to java.lang.Integer]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ClassCastException: java.lang.String cannot be cast to java.lang.Integer
	at com.qainfotech.tap.training.resourceio.model.Individual.<init>(Individual.java:18)
	at com.qainfotech.tap.training.resourceio.TeamsJsonReader.getListOfIndividuals(TeamsJsonReader.java:70)
	at com.qainfotech.tap.training.resourceio.TeamsJsonReader.getListOfTeams(TeamsJsonReader.java:266)
	at com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest.Team_getActiveMembers_should_return_a_list_of_team_members_that_are_active(TeamsJsonReaderTest.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ClassCastException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Team_getActiveMembers_should_return_a_list_of_team_members_that_are_active -->
        <test-method status="FAIL" signature="Team_getInactiveMembers_should_return_a_list_of_individual_team_members_that_are_inactive()[pri:0, instance:com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest@643b1d11]" name="Team_getInactiveMembers_should_return_a_list_of_individual_team_members_that_are_inactive" duration-ms="5" started-at="2017-06-27T14:24:57Z" finished-at="2017-06-27T14:24:57Z">
          <exception class="java.lang.ClassCastException">
            <message>
              <![CDATA[java.lang.String cannot be cast to java.lang.Integer]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ClassCastException: java.lang.String cannot be cast to java.lang.Integer
	at com.qainfotech.tap.training.resourceio.model.Individual.<init>(Individual.java:18)
	at com.qainfotech.tap.training.resourceio.TeamsJsonReader.getListOfIndividuals(TeamsJsonReader.java:70)
	at com.qainfotech.tap.training.resourceio.TeamsJsonReader.getListOfTeams(TeamsJsonReader.java:266)
	at com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest.Team_getInactiveMembers_should_return_a_list_of_individual_team_members_that_are_inactive(TeamsJsonReaderTest.java:114)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ClassCastException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Team_getInactiveMembers_should_return_a_list_of_individual_team_members_that_are_inactive -->
        <test-method status="FAIL" signature="getIndividualById_should_return_Individual_object_matching_id()[pri:0, instance:com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest@643b1d11]" name="getIndividualById_should_return_Individual_object_matching_id" duration-ms="1" started-at="2017-06-27T14:24:57Z" finished-at="2017-06-27T14:24:57Z">
          <exception class="com.qainfotech.tap.training.resourceio.exceptions.ObjectNotFoundException">
            <message>
              <![CDATA[null Object with null=null not found]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.qainfotech.tap.training.resourceio.exceptions.ObjectNotFoundException: null Object with null=null not found
	at com.qainfotech.tap.training.resourceio.TeamsJsonReader.getIndividualById(TeamsJsonReader.java:107)
	at com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest.getIndividualById_should_return_Individual_object_matching_id(TeamsJsonReaderTest.java:61)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- com.qainfotech.tap.training.resourceio.exceptions.ObjectNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getIndividualById_should_return_Individual_object_matching_id -->
        <test-method status="PASS" signature="getIndividualById_should_throw_ObjectNotFoundException_for_incorrect_id()[pri:0, instance:com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest@643b1d11]" name="getIndividualById_should_throw_ObjectNotFoundException_for_incorrect_id" duration-ms="2" started-at="2017-06-27T14:24:57Z" finished-at="2017-06-27T14:24:57Z">
          <exception class="com.qainfotech.tap.training.resourceio.exceptions.ObjectNotFoundException">
            <message>
              <![CDATA[null Object with null=null not found]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.qainfotech.tap.training.resourceio.exceptions.ObjectNotFoundException: null Object with null=null not found
	at com.qainfotech.tap.training.resourceio.TeamsJsonReader.getIndividualById(TeamsJsonReader.java:107)
	at com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest.getIndividualById_should_throw_ObjectNotFoundException_for_incorrect_id(TeamsJsonReaderTest.java:69)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- com.qainfotech.tap.training.resourceio.exceptions.ObjectNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getIndividualById_should_throw_ObjectNotFoundException_for_incorrect_id -->
        <test-method status="FAIL" signature="getIndividualByName_should_return_Individual_object_matching_name()[pri:0, instance:com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest@643b1d11]" name="getIndividualByName_should_return_Individual_object_matching_name" duration-ms="1" started-at="2017-06-27T14:24:57Z" finished-at="2017-06-27T14:24:57Z">
          <exception class="com.qainfotech.tap.training.resourceio.exceptions.ObjectNotFoundException">
            <message>
              <![CDATA[null Object with null=null not found]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.qainfotech.tap.training.resourceio.exceptions.ObjectNotFoundException: null Object with null=null not found
	at com.qainfotech.tap.training.resourceio.TeamsJsonReader.getIndividualByName(TeamsJsonReader.java:131)
	at com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest.getIndividualByName_should_return_Individual_object_matching_name(TeamsJsonReaderTest.java:77)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- com.qainfotech.tap.training.resourceio.exceptions.ObjectNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getIndividualByName_should_return_Individual_object_matching_name -->
        <test-method status="PASS" signature="getIndividualByName_should_throw_ObjectNotFoundException_for_incorrect_name()[pri:0, instance:com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest@643b1d11]" name="getIndividualByName_should_throw_ObjectNotFoundException_for_incorrect_name" duration-ms="2" started-at="2017-06-27T14:24:57Z" finished-at="2017-06-27T14:24:57Z">
          <exception class="com.qainfotech.tap.training.resourceio.exceptions.ObjectNotFoundException">
            <message>
              <![CDATA[null Object with null=null not found]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.qainfotech.tap.training.resourceio.exceptions.ObjectNotFoundException: null Object with null=null not found
	at com.qainfotech.tap.training.resourceio.TeamsJsonReader.getIndividualByName(TeamsJsonReader.java:131)
	at com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest.getIndividualByName_should_throw_ObjectNotFoundException_for_incorrect_name(TeamsJsonReaderTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- com.qainfotech.tap.training.resourceio.exceptions.ObjectNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getIndividualByName_should_throw_ObjectNotFoundException_for_incorrect_name -->
        <test-method status="FAIL" signature="getListOfActiveIndividuals_ahould_return_array_list_of_only_active_individual_objects()[pri:0, instance:com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest@643b1d11]" name="getListOfActiveIndividuals_ahould_return_array_list_of_only_active_individual_objects" duration-ms="2" started-at="2017-06-27T14:24:57Z" finished-at="2017-06-27T14:24:57Z">
          <exception class="java.lang.ClassCastException">
            <message>
              <![CDATA[java.lang.String cannot be cast to java.lang.Integer]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ClassCastException: java.lang.String cannot be cast to java.lang.Integer
	at com.qainfotech.tap.training.resourceio.model.Individual.<init>(Individual.java:18)
	at com.qainfotech.tap.training.resourceio.TeamsJsonReader.getListOfActiveIndividuals(TeamsJsonReader.java:230)
	at com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest.getListOfActiveIndividuals_ahould_return_array_list_of_only_active_individual_objects(TeamsJsonReaderTest.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ClassCastException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getListOfActiveIndividuals_ahould_return_array_list_of_only_active_individual_objects -->
        <test-method status="FAIL" signature="getListOfInactiveIndividuals_should_return_array_list_of_only_inactive_individual_objects()[pri:0, instance:com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest@643b1d11]" name="getListOfInactiveIndividuals_should_return_array_list_of_only_inactive_individual_objects" duration-ms="2" started-at="2017-06-27T14:24:57Z" finished-at="2017-06-27T14:24:57Z">
          <exception class="java.lang.ClassCastException">
            <message>
              <![CDATA[java.lang.String cannot be cast to java.lang.Integer]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ClassCastException: java.lang.String cannot be cast to java.lang.Integer
	at com.qainfotech.tap.training.resourceio.model.Individual.<init>(Individual.java:18)
	at com.qainfotech.tap.training.resourceio.TeamsJsonReader.getListOfInactiveIndividuals(TeamsJsonReader.java:177)
	at com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest.getListOfInactiveIndividuals_should_return_array_list_of_only_inactive_individual_objects(TeamsJsonReaderTest.java:50)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ClassCastException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getListOfInactiveIndividuals_should_return_array_list_of_only_inactive_individual_objects -->
        <test-method status="FAIL" signature="getListOfIndividuals_should_return_array_list_of_all_individual_objects()[pri:0, instance:com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest@643b1d11]" name="getListOfIndividuals_should_return_array_list_of_all_individual_objects" duration-ms="2" started-at="2017-06-27T14:24:57Z" finished-at="2017-06-27T14:24:57Z">
          <exception class="java.lang.ClassCastException">
            <message>
              <![CDATA[java.lang.String cannot be cast to java.lang.Integer]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ClassCastException: java.lang.String cannot be cast to java.lang.Integer
	at com.qainfotech.tap.training.resourceio.model.Individual.<init>(Individual.java:18)
	at com.qainfotech.tap.training.resourceio.TeamsJsonReader.getListOfIndividuals(TeamsJsonReader.java:70)
	at com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest.getListOfIndividuals_should_return_array_list_of_all_individual_objects(TeamsJsonReaderTest.java:27)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ClassCastException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getListOfIndividuals_should_return_array_list_of_all_individual_objects -->
        <test-method status="FAIL" signature="getListOfTeams_should_return_a_list_if_Team_object_from_db_json()[pri:0, instance:com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest@643b1d11]" name="getListOfTeams_should_return_a_list_if_Team_object_from_db_json" duration-ms="7" started-at="2017-06-27T14:24:57Z" finished-at="2017-06-27T14:24:57Z">
          <exception class="java.lang.ClassCastException">
            <message>
              <![CDATA[java.lang.String cannot be cast to java.lang.Integer]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ClassCastException: java.lang.String cannot be cast to java.lang.Integer
	at com.qainfotech.tap.training.resourceio.model.Individual.<init>(Individual.java:18)
	at com.qainfotech.tap.training.resourceio.TeamsJsonReader.getListOfIndividuals(TeamsJsonReader.java:70)
	at com.qainfotech.tap.training.resourceio.TeamsJsonReader.getListOfTeams(TeamsJsonReader.java:266)
	at com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest.getListOfTeams_should_return_a_list_if_Team_object_from_db_json(TeamsJsonReaderTest.java:91)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ClassCastException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getListOfTeams_should_return_a_list_if_Team_object_from_db_json -->
      </class> <!-- com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest -->
      <class name="com.qainfotech.tap.training.resourceio.PropertiesOptionsIOTest">
        <test-method status="PASS" signature="loadOptions()[pri:0, instance:com.qainfotech.tap.training.resourceio.PropertiesOptionsIOTest@100fc185]" name="loadOptions" is-config="true" duration-ms="10" started-at="2017-06-27T14:24:57Z" finished-at="2017-06-27T14:24:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loadOptions -->
        <test-method status="PASS" signature="loadDB()[pri:0, instance:com.qainfotech.tap.training.resourceio.TeamsJsonReaderTest@643b1d11]" name="loadDB" is-config="true" duration-ms="5" started-at="2017-06-27T14:24:57Z" finished-at="2017-06-27T14:24:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loadDB -->
        <test-method status="PASS" signature="addOptionShouldMethodShouldAppendPropertiesFileWithNewProperty()[pri:0, instance:com.qainfotech.tap.training.resourceio.PropertiesOptionsIOTest@100fc185]" name="addOptionShouldMethodShouldAppendPropertiesFileWithNewProperty" duration-ms="72" started-at="2017-06-27T14:24:57Z" finished-at="2017-06-27T14:24:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addOptionShouldMethodShouldAppendPropertiesFileWithNewProperty -->
        <test-method status="FAIL" signature="getOptionValueMethodShouldReturnOptionValuesFromPropertiesFile()[pri:0, instance:com.qainfotech.tap.training.resourceio.PropertiesOptionsIOTest@100fc185]" name="getOptionValueMethodShouldReturnOptionValuesFromPropertiesFile" duration-ms="19" started-at="2017-06-27T14:24:57Z" finished-at="2017-06-27T14:24:57Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<...pertiesOptionsIOTest[]"> but was:<...pertiesOptionsIOTest[#Thu Jun 22 12:47:56 IST 2017]">]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<...pertiesOptionsIOTest[]"> but was:<...pertiesOptionsIOTest[#Thu Jun 22 12:47:56 IST 2017]">
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at com.qainfotech.tap.training.resourceio.PropertiesOptionsIOTest.getOptionValueMethodShouldReturnOptionValuesFromPropertiesFile(PropertiesOptionsIOTest.java:29)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getOptionValueMethodShouldReturnOptionValuesFromPropertiesFile -->
      </class> <!-- com.qainfotech.tap.training.resourceio.PropertiesOptionsIOTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
